en/streaming-integrator/docs/quick-start-guide/quick-start-guide-101.md

```diff
@@ -224,13 +225,9 @@ To deploy your Siddhi application in the Streaming Integrator server, follow the
 !!!info
     To deploy the Siddhi application, you need to run both the Streaming Integrator server and Streaming Integrator Tooling. The home directories of the Streaming Integrator server is referred to as `<SI_HOME>` and the home directory of Streaming Integrator Tooling is referred to as `<SI_TOOLING_HOME>`.
 
-1. Start the Streaming Integrator via one of the following methods depending on your operating system.
-
-   - On MacOS/Linux/CentOS, open a terminal and issue the following command:
-
-       `sudo wso2si`
-
-   - On windows, go to **Start Menu -> Programs -> WSO2 -> Enterprise Integrator**. This opens a terminal. Start Streaming Integrator profile.
+1. Start the Streaming Integrator server by navigating to the `<SI_HOME>/bin` directory from the CLI, and issuing one of the following commands:
```

Comment:
Shouldn't we ask to issue the appropriate command as per their os. One of the commands doesn't sound to be correct.

Reply:
Changed as specified.